src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -12,23 +12,22 @@ const SLIPPAGE_SETTINGS = {
 export function SwapSettingsSlippageInput({
   className,
 }: SwapSettingsSlippageInputReact) {
-  const { updateLifecycleStatus, lifecycleStatus } = useSwapContext();
-  const getMaxSlippage = useCallback(() => {
-    return lifecycleStatus.statusData.maxSlippage;
-  }, [lifecycleStatus.statusData]);
```

Comment:
Removed the getMaxSlippage function. useCallback is likely not providing many performance benefits here. Likely it's even slower than accessing `lifecycleStatus.statusData.maxSlippage` directly

Reply:
Nice